src/auto/context.rs

```diff
@@ -104,13 +104,16 @@ impl Context {
     pub fn list_families(&self) -> Vec<FontFamily> {
         unsafe {
             let mut families = ptr::null_mut();
-            let mut n_families = mem::uninitialized();
+            let mut n_families = mem::MaybeUninit::uninit();
             pango_sys::pango_context_list_families(
                 self.to_glib_none().0,
                 &mut families,
-                &mut n_families,
+                n_families.as_mut_ptr(),
             );
-            FromGlibContainer::from_glib_container_num(families, n_families as usize)
+            FromGlibContainer::from_glib_container_num(families, {
+                let n_families = n_families.assume_init();
```

Comment:
Why inside the call to `from_glib_container_num`? Not that it matters, seems just like this was more difficult to add to gir than the alternative?

Reply:
It's a special case. Normally we only need to do it after the function has been called, however, in `FromGlibContainer` and equivalents, it's done at the same time. This one is one of the rare that doesn't do both at once.

Reply:
This also can changed to one-liner: https://github.com/gtk-rs/gir/pull/806#discussion_r300872790
so only difference from previous is `.assume_init()`